docs/faq.md

```diff
@@ -4,27 +4,27 @@
 
 ### Product
 
-**What is the difference between eZ Platform and Enterprise Edition?**
+**What is the difference between eZ Platform and eZ Enterprise?**
```

Comment:
Official name is eZ Platform Enterprise Edition, so I would leave `Enterprise Edition` as it is or change to `eZ Platform Enterprise Edition`.